lib/pbench/test/unit/server/query_apis/test_namespace_and_rows.py

```diff
@@ -26,7 +26,7 @@ def _setup(self, client):
             pbench_endpoint="/datasets/namespace/iterations",
             elastic_endpoint="/_search",
             payload={"run_id": "random_md5_string1"},
-            use_index_from_metadata=True,
+            index_from_metadata="result-data-sample",
```

Comment:
Do we really want to provide `index_name` explicitly? I thought we are trying to find `index_name` through `run_id` and `root_index_name`

Reply:
We search the dataset's index map (discovered via run_id, although we should change to use dataset name) to find indices with the expected root index name.

This only changes how the test system gets that root index name.